tests/performance/README.md

```diff
@@ -8,13 +8,17 @@ In the [locust_files] folder are located the test files.
 
 ## Usage
 
-1. Generate a `.env` file with setting for your test. After running `make install-dev` you can execute your test script in python and set settings as arguments. Once your settings are validated you pipe them to `.env`. E.g. if your testscript is `locust_files/platform_ping_test.py` you could run
+1. All settings are passed to the locust container as environment variables in `.env`. To generate locust env vars, run `locust_settings.py` with appropriate settings and pipe it to `.env`:
 ```bash
-python locust_files/platform_ping_test.py --LOCUST_HOST=https://api.osparc-master.speag.com  \
---LOCUST_USERS=100 --LOCUST_RUN_TIME=0:10:00 --SC_USER_NAME=myname --SC_PASSWORD=mypassword > .env
+python locust_settings.py --LOCUST_HOST=https://api.osparc-master.speag.com  \
+--LOCUST_USERS=100 --LOCUST_RUN_TIME=0:10:00 > .env
```

Comment:
could this not be also a Makefile recipe?
```make config``` ?

Reply:
Yep, good point. I have added this and adapted the doc here.